### ISSUE TITLE
Integers/numbers between don't work in reverse?

### ISSUE THREAD

**Issue by JKGamerxD** [SkriptLang/Skript#97]
Hey Bensku, I'm try to said is a bug with the expression:

```
[(integers) (between|from) %number% (and|to) %number% %number% times
```

Don't accept numbers in a reverse count, example:

```
loop integers between 3 to 1:
 send "%loop-number%"
```

And this isn't sent.

**Comment by JKGamerxD** [comment]
None progress with this @bensku?

**Comment by bensku** [comment]
I'm quite slow in adding new features now when there are bugs. Pull requests are welcome :)

**Comment by supertassu** [comment]
Wasn't this fixed in #1199?

**Comment by bensku** [comment]
Not yet released outside of nightly builds.
